content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -63,6 +74,13 @@ The following bulk operations run as transactions:
 
 ## The <inlinecode>$transaction</inlinecode> API
 
+The `$transactions` API can be used in two ways:
+
+- sequential operations: pass an array of Prisma Client operations that will be executed sequentially
+- interactive transactions (in Preview): pass a function in which you can perform a random set of (interactive) transactions
```

Comment:
I think "a random set of (interactive) transactions" is not correct. It is one transaction, that can execute arbitrary code that can include multiple Prisma Client queries. In comparison to the sequential of list of client queries, it can have control flow and other, non Prisma Client query code though.

Reply:
Thanks Jan can you take a look at my rewrite?

Reply:
I think you changed it in another instance than what I commented on.

Both instances (both introduction and in this section about "in two ways" do not fully use the term "transaction" correctly yet. The Transaction is always around all the things in there. Inside you execute either Prisma Client queries or arbitrary code that can include prisma client queries.

Reply:
Thanks @janpio I reworded to focus on fact that is it is one function, that can then execute multiple transactions.